apps/server/src/infra/sync/strategy/tsp/tsp-fetch.service.spec.ts

```diff
@@ -100,10 +93,10 @@ describe(TspFetchService.name, () => {
 		});
 
 		const exportApiMock = createMock<ExportApiInterface>();
-		exportApiMock.exportSchuleList.mockResolvedValueOnce(responseSchools);
-		exportApiMock.exportLehrerList.mockResolvedValueOnce(responseTeachers);
-		exportApiMock.exportSchuelerList.mockResolvedValueOnce(responseStudents);
-		exportApiMock.exportKlasseList.mockResolvedValueOnce(responseClasses);
+		exportApiMock.exportSchuleList.mockResolvedValue(responseSchools);
```

Comment:
I thought we wanted to continue working with `mockResolvedValueOnce` instead of this one

Reply:
didnt know about a decision on that... do you remember any reasons for it?
The problem is, that mockResolvedValueOnce cant be overwritten. its basically "queueing" that value, and multiple mocks get returned one after the other, until there is none left and mockResolvedValue takes over. That means, for the testcase where I want it to return an error, I either cant use the setup function, or I would have to first call the function to "unwind" the mocks. And if there was a testcase that called the function multiple times, it would break down completely as without further instruction, the function would return undefined in this case.

On the other hand, I cant think of any disadvantages to using mockResolvedValue here, since everything is cleared after each test anyway.

Reply:
I guess in this case your way is actually better hence my recollection says that the `..Once` was meant for specific purpose where we want something specifically to only be returned once and not more often. So its fine, nevermind ^^